CONTRIBUTING.md

```diff
@@ -148,11 +148,11 @@ cd rustc
 ./x.py setup compiler
  # Now edit `config.toml` and under `[rust]` set `debug-assertions = true`.
 
-# Build a stage 1 rustc, and build the rustc libraries with that rustc.
+# Build a stage 2 rustc, and build the rustc libraries with that rustc.
 # This step can take 30 minutes or more.
-./x.py build --stage 1 compiler/rustc
+./x.py build --stage 2 compiler/rustc
 # If you change something, you can get a faster rebuild by doing
-./x.py build --keep-stage 0 --stage 1 compiler/rustc
+./x.py build --keep-stage 0 --stage 2 compiler/rustc
 # You may have to change the architecture in the next command
 rustup toolchain link stage1 build/x86_64-unknown-linux-gnu/stage1
```

Comment:
This and the next line also need updating.

Reply:
Actually I'm not sure if they do. After I did the `x build --stage 2` build, I tried `./miri build` again, still with my `/stage1` toolchain, and it built successfully. I'll try running `cargo clean` and then building again just to make sure.

Reply:
I think rustbuild has some weird behavior around the interaction between `--stage 2` and `stage1` builds. It's surprising altogether that Miri requires a `--stage 2` build.

Reply:
But I can update it just to be safe if you want, given the previous history of these instructions ;)

Reply:
When I completed a stage 1 build, doing the stage 2 build doesn't even do much any more... it just builds libstd (which takes 60s) and then copies a bunch of stuff. Most of stage 2 is a copy of stage 1.

But yeah I'd prefer to consistently use stage 2 here.

Reply:
I `cargo clean`ed and ran `./miri build` again, and it worked, but I'll change it to be consistent.